pkg/model/resource/resource_test.go

```diff
@@ -136,8 +108,6 @@ var _ = Describe("Resource", func() {
 			resource = options.NewResource(multiGroupConfig, true)
 			Expect(resource.Plural).To(Equal("helmswomen"))
 
```

Comment:
Minot NIT: there is an extra blank line here.

Reply:
solved